e2e/solve/test.go

```diff
@@ -2,35 +2,36 @@ package solve
 
```

Comment:
I would rename `test.go`  to `order_test.go` to better represent what is being tested.